### ISSUE TITLE
Ember 2.1 beta 1 is on builds page but does not exist

### ISSUE THREAD

**Issue by ameliadowns** [emberjs/website#2316]
At emberjs.com/builds, the newest build listed 2.1 beta 1, but every single link (including "Download", "Debug", "Live Tests", and "Changelog") returns a 404.

**Comment by rwjblue** [comment]
yeah, thats my fault. Sorry about that.  I'm still working on publishing 2.1.0-beta.1.
